### PR TITLE
test count message

### PR DESCRIPTION
**Pull request by Safron1ta2**

### CODE REVIEW

**Review by Tsibatyuk — APPROVED** [review]

File: src/test/java/Safron/Foxtrot/Tests/TestCount.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    @Test
+    public void testCount() {
+        FoxtrotHomePageHelper foxtrotHomePageHelper = new FoxtrotHomePageHelper(driver);
+        FoxtrotSearchResultPageHelper foxtrotSearchResultPageHelper = new FoxtrotSearchResultPageHelper(driver);
+        CatalogViewElements catalogView = new CatalogViewElements(driver);
+        FoxtrotMonitorPageElements foxtrotMonitorPageElements = new FoxtrotMonitorPageElements(driver);
+
+        openUrl("https://www.foxtrot.com.ua/uk");
+
```
Comment (Wolmeright) [inline src/test/java/Safron/Foxtrot/Tests/TestCount.java:21]: 21 - unnecessary line

File: src/test/java/Safron/Foxtrot/Tests/TestCount.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import Safron.Foxtrot.Pages.FoxtrotMonitorPageElements;
+import Safron.Foxtrot.Pages.FoxtrotSearchResultPageHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestCount extends TestInit {
+
+    @Test
+    public void testCount() {
+        FoxtrotHomePageHelper foxtrotHomePageHelper = new FoxtrotHomePageHelper(driver);
```
Comment (Wolmeright) [inline src/test/java/Safron/Foxtrot/Tests/TestCount.java:15]: 1 more line beetween 14 and 15 lines

**Review by Wolmeright — CHANGES_REQUESTED** [review]

File: src/test/java/Safron/Foxtrot/Tests/TestCount.java (hunk 1 above)
Comment (Safron1ta2) [reply src/test/java/Safron/Foxtrot/Tests/TestCount.java:21]: i think no

File: src/test/java/Safron/Foxtrot/Tests/TestCount.java (hunk 2 above)
Comment (Safron1ta2) [reply src/test/java/Safron/Foxtrot/Tests/TestCount.java:15]: fixed

**Review by Wolmeright — APPROVED** [review]

**Pull request merged**